eo.txt

```diff
@@ -1273,6 +1273,13 @@ server_packet(Chair, Reply)
 	char = 0
 }
 
+"Your character standing up from a chair"
```

Comment:
Is this used for other players too?

Reply:
My understanding is that `SIT_REMOVE` is used for other players.
That's the behavior in EOSERV, I have never confirmed it in GameServer.

Reply:
I agree it's suspicious that `player_id` is in this packet, though.